examples/app_commands/basic.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+
+import discord
+from discord import app_commands
+
+MY_GUILD = discord.Object(id=0)  # replace with your guild id
+
+
+class MyClient(discord.Client):
+    def __init__(self, *, intents: discord.Intents, application_id: int):
+        super().__init__(intents=intents, application_id=application_id)
+
+        self.tree = app_commands.CommandTree(self)
+
+    # In this basic example, we just synchronize the app commands to one guild,
+    # without requiring to set the guild to every command individually by just copying them over.
+    # By doing so we don't have to wait up to an hour until they are shown to the end-user.
+    async def setup_hook(self):
+        # This copies the global commands over to your guild.
+        self.tree.copy_global_to(guild=MY_GUILD)
+        await self.tree.sync(guild=MY_GUILD)
+
+
+intents = discord.Intents.default()
+
+# In order to use a basic synchronization of the app commands in the setup_hook,
+# you have replace ... with your bots application_id you find in the developer portal.
+client = MyClient(intents=intents, application_id=0)
+
+
+@client.event
+async def on_ready():
+    print(f'Logged in as {client.user} (ID: {client.user.id})')
+    print('------')
+
+
+@client.tree.command()
+async def hello(interaction: discord.Interaction):
+    """Says hello!"""
+    await interaction.response.send_message(f'Hi, {interaction.user.mention}')
+
+
+@client.tree.command()
+@app_commands.describe(
+    first_value='The first value you want to add something to', second_value='The value you want to add to the first value'
```

Comment:
```suggestion
    first_value='The first value you want to add something to', 
    second_value='The value you want to add to the first value',
```

Magic commas are better for readability.